### PR TITLE
Updated manifests for replication 1.7.0

### PR DESCRIPTION
**Pull request by tdawe**
# Description
Updated manifests for replication 1.7.0

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/1012 |

# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have maintained backward compatibility

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [x] Built operator, deployed into cluster. Created new CSM CR for PowerScale with Replication enabled. Verified the pods were created for both the driver and module. Pods were unhealthy due to images not yet existing on Docker Hub.

### CODE REVIEW

**Review by KerryKovacevic — APPROVED** [review]

File: bundle/manifests/dell-csm-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -738,7 +738,7 @@ metadata:
                         "value": "powermax"
                       }
                     ],
-                    "image": "dellemc/dell-csi-replicator:v1.6.0",
+                    "image": "dellemc/dell-csi-replicator:v1.7.0",
```
Comment (santhoshatdell) [inline bundle/manifests/dell-csm-operator.clusterserviceversion.yaml:741]: Could you make similar changes for powerscale and powerflex in this file?

File: operatorconfig/moduleconfig/replication/v1.7.0/container.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+name: dell-csi-replicator
```
Comment (santhoshatdell) [inline operatorconfig/moduleconfig/replication/v1.7.0/container.yaml:1]: Could you remove 1.4.0 version replication files as we support n-2 versions in CSM Operator?

operatorconfig/moduleconfig/common/version-values.yaml also needs update.

Comment (tdawe) [reply operatorconfig/moduleconfig/replication/v1.7.0/container.yaml:1]: Done

File: bundle/manifests/dell-csm-operator.clusterserviceversion.yaml (hunk 1 above)
Comment (tdawe) [reply bundle/manifests/dell-csm-operator.clusterserviceversion.yaml:741]: Done

**Review by alikdell — APPROVED** [review]

**Review by JacobGros — APPROVED** [review]

**Pull request merged**